lib/jira.ts

```diff
@@ -193,8 +183,16 @@ export class Jira {
         return true;
     }
 
-    public static getFinalStatusNames(config: ProjectConfig): string[] {
-        return config.statuses.filter(status => status.isDone).map(status => status.name);
+    public static getFinalStatuses(statuses: Status[]): Status[] {
+        return statuses.filter(status => status.isDone);
+    }
+
+    public static isDoneStatus(statuses: Status[], statusToCheck: string): Boolean {
```

Comment:
Should this be called "isFinalStatus" instead? I've tried not to use the "Done" word, since it's a common Jira status name, and I tried to be more generic with that. It sounds to me that both should be named the same, be it "Final" or "Done"...

Reply:
I was a little bit confused about the usage of "Final" and "Done".  "Done" is quite commonly used for something that is ready e.g. Definition of Done. "Final" sounds to me like "the last one" or end state of a state machine. That's clearly not the case here since we prepare for state changes also after a final state.

Jira has three categories for states: To Do, In Progress and Done. So my vote goes for Done. And I agree that only one of those should be used.